CHANGELOG.rst

```diff
@@ -1,6 +1,30 @@
 Changelog
 ==========
 
+.. _2.8.0:
+
+2.8.0 - 2024-09-24
+~~~~~~~~~~~~~~~~~~~
+- New features:
+    - Add CODEOWNERS file to the repository.
```

Comment:
Isn't this under the "workflow (e.g. GitHub Actions) etc" category, which shouldn't be added?

Reply:
I meant that instructions say 
`Do not add information regarding workflow (e.g. GitHub Actions) etc`

Reply:
Ah sorry, I thought that part was for the release notes, I didn't remember that